src/extension/features/budget/resizable-columns/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'ResizableColumns',
+  type: 'checkbox',
+  default: false,
+  section: 'budget',
+  title: 'Resizable Columns',
+  description: `Allows clicking/right-clicking the budget table headers to grow/shrink the width of the number columns. Middle-click to reset.`,
```

Comment:
I want to say a large majority of OSX users (myself) use a trackpad, we should find a trackpad-friendly way to implement this feature.

Reply:
Two-finger click?

Reply:
two-finger click is how you'd shrink, middle click is used to reset but I'm not sure there's a great way to do that with a trackpad?